### ISSUE TITLE
Temporarily ignore/disable/undo plain text emoji replacement

### ISSUE THREAD

**Issue by jthomas281** [vector-im/element-web#22008]
### Steps to reproduce

1. Where are you starting? What can you see?
2. What do you click?
3. More steps…

#### What did you expect?
The letters "XP" to appear (like Windows XP)

#### What happened instead?
Got an emoji. No way to escape it.

### Will you send logs?

No

**Comment by jthomas281** [comment]
Looks like I can escape it with /plain or /html but still - having common letters automatically transformed into an emoji is probably not a good idea.

**Comment by robintown** [comment]
This would be because you've enabled the "Automatically replace plain text emoji" option

**Comment by jthomas281** [comment]
Thanks, it is enabled though I don't recall having ever enabled it. Turned it off. Nevertheless, my point still stands. XP is a more common sequence for an abbreviation than an emoji (which I almost never see anyone use) and I don't think it should be automatically converted or there should be an option to "undo" any emoji conversion (perhaps with a backspace).

**Comment by robintown** [comment]
Make sense, I've turned this into an issue to allow undoing/ignoring the suggestion.

**Comment by kerryarchibald** [comment]
Duplicate https://github.com/vector-im/element-web/issues/19472

**Comment by robintown** [comment]
I don't think this is a duplicate, the only proposed solution at #19472 is to not do the conversion unless there's a space before the emoticon, but that would still leave "Windows XP" converted to "Windows :stuck_out_tongue_closed_eyes:" with no way to avoid/undo that
